### PR TITLE
Feature/api 22 command set

### PR DESCRIPTION
**Pull request by renemarc**
Solution for #6

### CODE REVIEW

File: lightpack.py [hunk 1]
```diff
@@ -14,12 +14,12 @@
 DESCRIPTION = "Library to control Lightpack"
 AUTHOR = "Bart Nagel <bart@tremby.net>, Mikhail Sannikov <atarity@gmail.com>"
 URL = 'https://github.com/tremby/py-lightpack'
-VERSION = '2.3.0'
+VERSION = '2.1.0'
```
Comment (tremby) [inline lightpack.py]: The version number is dropping here.

File: lightpack.py [hunk 2]
```diff
@@ -824,6 +987,15 @@ def __init__(self, command, response, expected):
 		self.command = command
 		self.response = response
 		self.expected = expected
+class CommandDeprecatedError(RuntimeError):
```
Comment (tremby) [inline lightpack.py:972]: Space between classes

**Review by tremby — CHANGES_REQUESTED** [review]

File: lightpack.py (hunk 2 above)
Comment (renemarc) [reply lightpack.py:972]: Fixed!

**Comment by renemarc** [comment]
Rebased on **feature/api-15-command-set** to minimize merge conflicts.

File: README.md [hunk 3]
```diff
@@ -92,7 +92,10 @@ except lightpack.CannotConnectError as e:
 # Read the current states
 print("Status:           %s" % lp.getStatus())
 print("API status:       %s" % lp.getApiStatus())
+<<<<<<< HEAD
 print("API version:      %s" % lp.getApiVersion())
+=======
+>>>>>>> Implement all API v1.4 commands
 print("Locked:           %s" % lp.getLockStatus())
```
Comment (tremby) [inline README.md:96]: There's a rebase mistake here, which you fixed later. I'm going to tidy up a little and merge.

**Comment by tremby** [comment]
I've manually merged and rebased these changes. Thanks very much!

**Pull request closed without merging**